src/pug/blog/typing_methods.pug

```diff
@@ -0,0 +1,425 @@
+extends _blog.pug
+
+block append subtitle
+  | Alphanumeric input
+
+block content
+  h2 Alphanumeric input
+  hr
+  p
+    i 27 March 2023 <br>
+  hr
+
+  h3 1. Introduction
+  p In some scenarios it is necessary to input alphanumeric characters while using a device without a physical keyboard (mobile phone, videogame console, smart TV) or when using a keyboard is not possible (disability use cases). Solutions for these cases have been created in the past:
+  p #[+ax('https://en.wikipedia.org/wiki/Feature_phone') Feature phones] used keypad typing #[i (Fig. A)].
+  p Smartphones uses touchscreen keyboards #[i (Fig. B)].
+  p Videogame consoles and computers uses on-screen virtual keyboards #[i (Fig. C)].
+  +figure(
+    'A',
+    'Nokia 3210 keypad (1999)',
+    '/static/blog/keypad.jpg',
+    'small inline'
+  )
+  +figure(
+    'B',
+    'Apple iPhone 3G touchscreen keyboard (2008)',
+    '/static/blog/iphone3g_keyboard.png',
+    'small inline'
+  )
+  //- +figure(
+  //-   'C',
+  //-   'Sony PlayStation 4 keyboard (2013)',
+  //-   '/static/blog/ps4_keyboard.jpg',
+  //-   'small inline'
+  //- )
+  +figure(
+    'C',
+    'Microsoft Windows 10 on-screen keyboard (2015)',
+    '/static/blog/win_keyboard.jpg',
+    'big inline'
+  )
+
+  p In the context of a computer dedicated to videogames, or a videogame console, the most frequent use cases for alphanumeric input are:
```

Comment:
Why table?
Better: "the following use-cases:
1. Search box: input in a game launcher ..."